src/util/Util.jl

```diff
@@ -214,5 +214,94 @@ function point_on_limb(ellipsoid, observer, outside)
     return distance(t1, outside) <= distance(t2, outside) ? t1 : t2
 end
 
+
+"""
+spherical_to_cartesian(theta, phi)
+
+Convert spherical coordinates to Cartesian.
+
+### Arguments ###
+
+- `theta`: longitude angle in radians
+- `phi`: latitude angle in radians
+
+### Returns ###
+- `x`: magnitude of projection on x axis
+- `y`: magnitude of projection on y axis
+- `z`: magnitude of projection on z axis
+
+Derived, with permission, from the SOFA library.  See notes at end of file.
+"""
+function spherical_to_cartesian(theta, phi)
+   cp = cos(phi);
+   x = cos(theta) * cp;
+   y = sin(theta) * cp;
+   z = sin(phi);
+
+   return x, y, z
+end
+
+
+"""
```

Comment:
It should suffice to put this into the LICENSE.md and just put a reference here, no? IANAL and all.

Reply:
No clue. Should I do it?

Reply:
Just remove it for now. I will take care of updating the LICENSE file on `master`.

Reply:
I have asked the ERFA maintainers how we should handle this:https://github.com/liberfa/erfa/issues/67